packages/retail-ui-extensions/src/extension-api/scanner-api/scanner-api.ts

```diff
@@ -0,0 +1,30 @@
+import type {RemoteSubscribable} from '@remote-ui/async-subscription';
+
+/** The scanner source the POS device supports. */
+export type ScannerSource = 'camera' | 'external' | 'embedded';
+
+export interface ScannerSubscriptionResult {
+  /** The string data from the last scanner event received. */
+  data?: string;
+  /** The scanning source from which the scan event came. */
+  scannerSource: ScannerSource;
```

Comment:
```suggestion
  source: ScannerSource;
```
How I see it, is its implied that its a scanner source if its inside the ScannerSubscriptionResult. If we follow this flow, why isn't data named scannerData?